test-app/tests/helpers/dom.ts

```diff
@@ -0,0 +1,24 @@
+import { find, triggerEvent } from '@ember/test-helpers';
+
+/**
+ * Fill the provided text into the `value` property of the selected form element, similar to `fillIn`, but *without* implicitly triggering a `change` event.
+ * This mimics the behavior of a user typing data into an input without yet focusing out of it. Browsers will only trigger a `change` event when focusing
+ * out of the element, not while typing!
+ *
+ * `fillIn` will basically simulate entering the data *and* kinda focusing out (as it triggers `change`, however not `focusout`, which is impossible to achieve as a real user),
```

Comment:
😮 I did NOT know this!

Reply:
TIL

Reply:
Yeah, had to confirm this for myself by looking up [their source](https://github.com/emberjs/ember-test-helpers/blob/master/addon-test-support/%40ember/test-helpers/dom/fill-in.ts#L78-L80).

I should probably create an issue. I guess we cannot change the existing behaviour of `fillIn`, as that would break _every_ app. But maybe we can upstream some additional helpers that cover the different real-world use cases better...

Reply:
Filed this: https://github.com/emberjs/ember-test-helpers/issues/1336